frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.js

```diff
@@ -0,0 +1,292 @@
+        apiJob = await jobApiCall(selectedColorLayerName);
+      }
+      if (!apiJob) {
+        return;
+      }
+      Toast.info(
+        <>
+          The {jobName} job has been started. You can look in the{" "}
+          <a target="_blank" href="/jobs" rel="noopener noreferrer">
+            Processing Jobs
+          </a>{" "}
+          view under Administration for details on the progress of this job.
+        </>,
+      );
+      handleClose();
+    } catch (error) {
+      console.error(error);
+      Toast.error(
+        `The ${jobName} job could not be started. Please contact an administrator or look in the console for more details.`,
+      );
+      handleClose();
+    }
+  };
+
+  const ColorLayerSelection = (): Node =>
+    colorLayerNames.length > 1 ? (
+      <React.Fragment>
+        <p>
+          The detection approach uses a single color layer. Please select the layer that should be
+          used for detection.
+        </p>
+        <div style={{ textAlign: "center" }}>
+          <Select
+            showSearch
+            style={{ width: 300 }}
+            placeholder="Select a color layer"
+            optionFilterProp="children"
+            value={selectedColorLayerName}
+            onChange={setSelectedColorLayerName}
+            filterOption={(input, option) =>
+              option.children.toLowerCase().indexOf(input.toLowerCase()) >= 0
+            }
+          >
+            {colorLayerNames.map(colorLayerName => (
+              <Select.Option key={colorLayerName} value={colorLayerName}>
+                {colorLayerName}
+              </Select.Option>
+            ))}
+          </Select>
+        </div>
+        <br />
+      </React.Fragment>
+    ) : null;
+
+  const renderUserBoundingBox = (bbox: ?UserBoundingBox) => {
+    if (!bbox) {
+      return null;
+    }
+    const upscaledColor = ((bbox.color.map(colorPart => colorPart * 255): any): Vector3);
+    const colorAsHexString = rgbToHex(upscaledColor);
+    return (
+      <>
+        {bbox.name}{" "}
+        <div
+          className="color-display-wrapper"
+          style={{
+            backgroundColor: colorAsHexString,
+            marginTop: -2,
+          }}
+        />{" "}
+        - {computeArrayFromBoundingBox(bbox.boundingBox).join(", ")}
+      </>
+    );
+  };
+  const BoundingBoxSelection = (): Node =>
+    isBoundingBoxConfigurable ? (
+      <React.Fragment>
+        <p>
+          The detection requires you to set a bounding box for the volume where the detection shall
+          be done. As the detection takes a long time, it is suggested to only take a small volume
+          and not the whole dataset. To select a volume please create a bounding box of the desired
+          volume with the bounding box tool in the toolbar at the top. The created bounding boxes
+          will be available below for selection. The format of the Bounding Boxes is &quot;minX,
+          minY, minZ, width, height, depth&quot;.
```

Comment:
I tried to make it a bit shorter:
```suggestion
          Please select the bounding box for which the inferral should be computed. Note that large
          bounding boxes, can take very long.
          You can create a new bounding box for the desired volume with the bounding box tool in the toolbar at the top. 
          The created bounding boxes will be listed below. The format of the Bounding Boxes is &quot;minX,
          minY, minZ, width, height, depth&quot;.
```

Reply:
I think it would be cool allow inputting a bounding box in this modal directly. This is what I initially tried before reading the wall of text. But that would be more coding so let’s defer that maybe.

Philipp’s suggestion has a superfluous comma in `bounding boxes, can take very long`.
I’d suggest omitting the format hint at the end of the text here. If a user managed to create a bounding box, they just need to select it now. In fact, the format hint was the first thing I saw here so I immediately started typing.